src/sos.jl

```diff
@@ -43,7 +43,7 @@ end
 
 addSOS1(m::Model, coll) = addSOS1(m, convert(Vector{AffExpr}, coll))
 
-function addSOS1(m::Model, coll::Vector{AffExpr})
+function addSOS1(m::Model, coll::AbstractVector{AffExpr})
```

Comment:
Not covered by tests. This method calls ``constructSOS`` which is defined for ``Vector{AffExpr}``.